test_gpios.py

```diff
@@ -58,15 +58,19 @@ async def test_interrupts(self):
         DURATION = 2 # seconds
         ERROR_FACTOR = 0.05
 
-        await self.pwm_pin.set(False) # Turn the output off
-        starting_count = await self.interrupt.value()
-
         await self.pwm_pin.set_pwm_frequency(FREQUENCY)
         await self.pwm_pin.set_pwm(0.5) # Duty cycle fraction: 0 to 1
+
+        # Sometimes a board has a small delay between when you set the PWM and
+        # when the signal starts outputting. To hopefully compensate for this,
+        # we'll wait a shart amount before we start counting cycles.
+        await asyncio.sleep(0.5)
```

Comment:
Is half a second too much delay? I think it's perfectly fine for a 2am check-in - but I wanted to ask the question.

Reply:
I totally agree! It's fine for a 2:30 AM check, but I wouldn't want to do this while a human was waiting on the results. I have noticed boards that don't turn on the output until I think it's been too long, which I suspect is hundreds of milliseconds. It would be nice to have measurements on this, but obtaining them sounds super boring.